.zuul.d/jobs.yaml

```diff
@@ -16,3 +16,5 @@
     vars:
       tox_envlist: integration
       tox_install_siblings: false
+      tox_environment:
+        KEEP_IMAGES: true
```

Comment:
Should this be reversed to `false`? We are running an ephemeral container, we do not want to keep images because it won't matter.

Reply:
no, I was wrong, never mind